src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -54,6 +72,32 @@ private static CloudBillingClient cloudBillingClient() {
         }
     }
 
+    private static CloudBillingClient cloudBillingClient() {
+        return cloudBillingClient(null);
+    }
+
+    public static boolean verifyAccess(AuthenticatedUserRequest user, String billingAccountId) {
+        ResourceName resource = BillingAccountName.of(billingAccountId);
+        List<String> permissions = Collections.singletonList("billing.resourceAssociations.create");
+        TestIamPermissionsRequest permissionsRequest = TestIamPermissionsRequest.newBuilder()
+            .setResource(resource.toString())
+            .addAllPermissions(permissions)
```

Comment:
Since it's one permission you can write
```suggestion
            .addPermissions("billing.resourceAssociations.create")
```

Reply:
That adds a bit of complexity for checking the response as it returns a list of permissions.